eb_hooks.py

```diff
@@ -404,6 +401,39 @@ def pre_single_extension_isoband(ext, *args, **kwargs):
         ext.cfg['preinstallopts'] = "sed -i 's/SIGSTKSZ/32768/g' src/testthat/vendor/catch.h && "
 
 
+def pre_single_extension_numpy(ext, *args, **kwargs):
+    """
+    Pre-extension hook for numpy, to change -march=native to -march=armv8.4-a for scipy 1.10.x
```

Comment:
this still says `scipy`, should be `numpy`?

Reply:
@bedroge fixed in 937efa6